<launch>

	<node pkg="classic_robot" type="turtle_teleop_joy" name="teleop"/>

</launch>